src/Plugin.php

```diff
@@ -2,14 +2,19 @@
 declare(strict_types=1);
 
 /**
- * This file is part of TwigView.
+ * CakePHP(tm) : Rapid Development Framework (https://cakephp.org)
+ * Copyright (c) Cake Software Foundation, Inc. (https://cakefoundation.org)
```

Comment:
Should we have the date of transition as well?

Reply:
`Original work Copyright ...`
`Modified work Copyright ...`
?

Reply:
Could do `2020-present`

Reply:
I don't think you can claim continuous copyright. I think it should be dated or undated.

Reply:
Ok. No dates on both is how we did the carbon -> chronos transition.

Reply:
I'm authority on this, but I think the original copyright date should be left and cake's can be undated.

It's really semantics for code copyright because the small difference means nothing compared to when the copyright expires vs when the code is useful.

I just think we probably shouldn't restrict his original copyright.